### PR TITLE
[T3A3][W10-A4]Moon Byunghun

### PR DESCRIPTION
**Pull request by MoonByunghun**
Splitting Address class into more classes.

### CODE REVIEW

**Comment by MoonByunghun** [comment]
Ready for review.

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -8,11 +8,16 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
+    public static final String EXAMPLE = "unit-no, block-no, some street, postal code";
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Address.java]: This is not a good example :-p It doesn't even look like an address :-)

File: src/seedu/addressbook/data/person/Address.java [hunk 2]
```diff
@@ -25,8 +30,33 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
...
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
+        String[] addressDetails = address.split(",");
+        if(addressDetails.length!=4){
+        	throw new IllegalValueException(MESSAGE_ADDRESS_WRONG_FORMAT);
+        }
+        unit = new Unit(addressDetails[0].trim());
+        block = new Block(addressDetails[1].trim());
+        street = new Street(addressDetails[1].trim());
+        postalcode = new PostalCode(addressDetails[1].trim());
```
Comment (damithc) [inline src/seedu/addressbook/data/person/Address.java]: Doesn't look correct. Why the same value is used thrice? To prevent tests breaking? :question:

**Comment by damithc** [comment]
Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**